R/zchunk_LA124.heat.R

```diff
@@ -69,117 +77,121 @@ module_energy_LA124.heat <- function(command, ...) {
       group_by(technology, subsector, supplysector, minicam.energy.input)  %>%
       mutate(value = approx_fun(year, value)) %>%
       left_join(distinct(calibrated_techs)) %>%
-      select(-calibration, -secondary.output) -> L124.globaltech_coef
+      select(supplysector, subsector, technology, minicam.energy.input, year, value, sector, fuel) -> L124.globaltech_coef
 
     # Heat output: fuel inputs to heat divided by exogenous input-output coefficients
     L124.in_EJ_R_heat_F_Yh %>%
       left_join(L124.globaltech_coef %>%
-                  rename(temp=value),
-                      by=c("sector", "fuel", "year")) %>%
-      mutate(value=value/temp)%>%
+                  rename(temp = value),
+                      by = c("sector", "fuel", "year")) %>%
+      mutate(value = value / temp) %>%
```

Comment:
Bit confusing here with temp and value with the nested structure. You could rename temp to temp_IOCoef for clarity.